qemu/tests/cfg/hpt.cfg

```
@@ -20,6 +20,7 @@
             maxmem_mem = 64G
             slots_mem = 32
             size = "1073741824"
+            wait_time = 20
```

If only ppc hit this issue, better to set unless x86 agrees:
```suggestion
            ppc64, ppc64le:
                wait_time = 20
```